shogun-lib/pom.xml

```diff
@@ -19,11 +19,18 @@
         <groupId>org.apache.maven.plugins</groupId>
         <artifactId>maven-compiler-plugin</artifactId>
       </plugin>
+      <plugin>
+        <groupId>com.google.cloud.tools</groupId>
+        <artifactId>jib-maven-plugin</artifactId>
+        <configuration>
+          <!-- we don't want jib to execute on this module since its a library -->
+          <skip>true</skip>
+        </configuration>
+      </plugin>
```

Comment:
See above.

Reply:
I've removed the unneeded config in 915aa882